File: Code/Components/DanpheEMR.ServerModel/ClinicalModels/PrescriptionSlip/PrescriptionSlipModel.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DanpheEMR.ServerModel
{
    public class PrescriptionSlipModel
    {
        [Key]
        public int Id { get; set; }
        public int VisitId { get; set; }
        public int ProviderId { get; set; }
        public int PatientId { get; set; }
        public DateTime VisitDate { get; set; }
        public DateTime ModifiedOn { get; set; }
        public int ModifiedBy { get; set; }
        public int CreatedBy { get; set; }
        public DateTime CreatedOn { get; set; }

        [NotMapped]
        public AcceptanceModel Acceptance { get; set; }
        [NotMapped]
        public HistoryModel History { get; set; }
        [NotMapped]
        public DilateModel Dilate { get; set; }
        [NotMapped]
        public IOPModel IOP{ get; set; }
        [NotMapped]
        public PlupModel Plup { get; set; }
        [NotMapped]
        public RetinoscopyModel Retinoscopy { get; set; }
        [NotMapped]
        public SchrimeModel Schrime { get; set; }
        [NotMapped]
        public TBUTModel TBUT { get; set; }
        [NotMapped]
        public VaUnaidedModel VaUnaided{ get; set; }
        [NotMapped]
        public FinalClassModel FinalClass { get; set; }
        [NotMapped]
        public AdviceDiagnosisModel AdviceDiagnosis { get; set; }
    }
}

File: Code/Websites/DanpheEMR/wwwroot/DanpheApp/src/app/clinical/eye-examination/prescription-slip/shared/PrescriptionSlip.model.ts
import {
  NgForm,
  FormGroup,
  FormControl,
  Validators,
  FormBuilder,
  ReactiveFormsModule
} from '@angular/forms';
import * as moment from 'moment/moment';
import { HistoryModel } from './history.model';
import { PlupModel } from './plup.model';
import { VaUnaidedModel } from './va-unaided.model';
import { RetinoscopyModel } from './retinoscopy.model';
import { AcceptanceModel } from './acceptance.model';
import { SchrimeModel } from './schrime.model';
import { TBUTModel } from './TBUT.model';
import { DilateModel } from './dilate.model';
import { IOPModel } from './IOP.model';
import { FinalClassModel } from './FinalClass.model';
import { AdviceDiagnosisModel } from './advicediagnosis.model';

export class PrescriptionSlipModel {

  public SNo: number = 0;
  public VisitId: number = 0;
  public PatientId: number = 0;
  public ProviderId: number = 0;
  public MasterId: number = 0;
  public VisitDate: Date = new Date();
  public ModifiedOn: Date = new Date();
  public CreatedBy: number = 0;
  public CreatedOn: Date = new Date();


  public History: HistoryModel = new HistoryModel();
  public Plup: PlupModel = new PlupModel();
  public Retinoscopy: RetinoscopyModel = new RetinoscopyModel();
  public Acceptance: AcceptanceModel = new AcceptanceModel();
  public Dilate: DilateModel = new DilateModel();
  public IOP: IOPModel = new IOPModel();
  public Schrime: SchrimeModel = new SchrimeModel();
  public TBUT: TBUTModel = new TBUTModel();
  public VaUnaided: VaUnaidedModel = new VaUnaidedModel();
  public FinalClass: FinalClassModel = new FinalClassModel();
  public AdviceDiagnosis: AdviceDiagnosisModel = new AdviceDiagnosisModel();

  constructor() {
  }



}
